### ISSUE TITLE
Error in rule call_integrate_sources

### ISSUE THREAD

**Issue by HideyukiMiyazawa** [EichlerLab/pav#37]
Hi. I have used PAV several times, but this time it doesn't work after pull request of PAV, and updating snakemake through conda, 

The end of log file (.snakemake/log/2023-06-02T114200.000215.snakemake.log) as follows: 

```
[Fri Jun  2 11:44:43 2023]
rule call_integrate_sources:
    input: temp/type1/cigar/pre_inv/svindel_insdel_h2.bed.gz, temp/type1/cigar/pre_inv/snv_snv_h2.bed.gz, temp/type1/lg_sv/sv_ins_h2.bed.gz, temp/type1/lg_sv/sv_del_h2.bed.gz, temp/type1/lg_sv/sv_inv_h2.bed.gz, temp/type1/inv_caller/sv_inv_h2.bed.gz
    output: results/type1/bed/pre_merge/h2/svindel_ins.bed.gz, results/type1/bed/pre_merge/h2/svindel_del.bed.gz, results/type1/bed/pre_merge/h2/sv_inv.bed.gz, results/type1/bed/pre_merge/h2/snv_snv.bed.gz, results/type1/bed/dropped/sv_inv_h2.bed.gz, results/type1/bed/dropped/svindel_insdel_h2.bed.gz, results/type1/bed/dropped/snv_snv_h2.bed.gz
    jobid: 105
    reason: Missing output files: results/type1/bed/pre_merge/h2/sv_inv.bed.gz, results/type1/bed/pre_merge/h2/snv_snv.bed.gz, results/type1/bed/pre_merge/h2/svindel_ins.bed.gz, results/type1/bed/pre_merge/h2/svindel_del.bed.gz; Input files updated by another job: temp/type1/inv_caller/sv_inv_h2.bed.gz, temp/type1/cigar/pre_inv/snv_snv_h2.bed.gz, temp/type1/cigar/pre_inv/svindel_insdel_h2.bed.gz, temp/type1/lg_sv/sv_ins_h2.bed.gz, temp/type1/lg_sv/sv_inv_h2.bed.gz, temp/type1/lg_sv/sv_del_h2.bed.gz
    wildcards: asm_name=type1, hap=h2
    resources: tmpdir=/tmp

[Fri Jun  2 11:44:43 2023]
rule call_mappable_bed:
    input: results/type1/align/aligned_tig_h2.bed.gz, temp/type1/lg_sv/sv_del_h2.bed.gz, temp/type1/lg_sv/sv_ins_h2.bed.gz, temp/type1/lg_sv/sv_inv_h2.bed.gz
    output: results/type1/callable/callable_regions_h2_500.bed.gz
    jobid: 202
    reason: Missing output files: results/type1/callable/callable_regions_h2_500.bed.gz; Input files updated by another job: results/type1/align/aligned_tig_h2.bed.gz, temp/type1/lg_sv/sv_ins_h2.bed.gz, temp/type1/lg_sv/sv_inv_h2.bed.gz, temp/type1/lg_sv/sv_del_h2.bed.gz
    wildcards: asm_name=type1, hap=h2, flank=500
    resources: tmpdir=/tmp

[Fri Jun  2 11:44:45 2023]
Finished job 202.
196 of 285 steps (69%) done
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-06-02T114200.000215.snakemake.log
```

Is there anything I can do? and, is this problem caused by the version of snakemake or PAV?

Thanks,
Hide

**Comment by HideyukiMiyazawa** [comment]
And the end of the displayed log as follows:

```[Fri Jun  2 11:44:44 2023]
Error in rule call_integrate_sources:
    jobid: 0
    input: temp/type1/cigar/pre_inv/svindel_insdel_h2.bed.gz, temp/type1/cigar/pre_inv/snv_snv_h2.bed.gz, temp/type1/lg_sv/sv_ins_h2.bed.gz, temp/type1/lg_sv/sv_del_h2.bed.gz, temp/type1/lg_sv/sv_inv_h2.bed.gz, temp/type1/inv_caller/sv_inv_h2.bed.gz                                
    output: results/type1/bed/pre_merge/h2/svindel_ins.bed.gz, results/type1/bed/pre_merge/h2/svindel_del.bed.gz, results/type1/bed/pre_merge/h2/sv_inv.bed.gz, results/type1/bed/pre_merge/h2/snv_snv.bed.gz, results/type1/bed/dropped/sv_inv_h2.bed.gz, results/type1/bed/dropped/svindel_insdel_h2.bed.gz, results/type1/bed/dropped/snv_snv_h2.bed.gz

RuleException:
AttributeError in file /home/myname/pav/rules/call.snakefile, line 466:
'DataFrame' object has no attribute 'append'
  File "/home/myname/pav/rules/call.snakefile", line 466, in __rule_call_integrate_sources
  File "/home/myname/anaconda3/envs/pav/lib/python3.11/site-packages/pandas/core/generic.py", line 5989, in __getattr__
  File "/home/myname/anaconda3/envs/pav/lib/python3.11/concurrent/futures/thread.py", line 58, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
[Fri Jun  2 11:44:45 2023]
Finished job 202.
196 of 285 steps (69%) done
Exiting because a job execution failed. Look above for error message
Complete log: .snakemake/log/2023-06-02T114200.000215.snakemake.log```

Thanks,

**Comment by HideyukiMiyazawa** [comment]
Thank you for reply.
I have finished the PAV analyses by reverting Pandas back to the old version by

`conda install -c conda-forge pandas==1.5.3 ;`

I saved a log of PAV's stdout and stderr outputs, which stated  

`/home/myname/pav/rules/call.snakefile:511: FutureWarning: The frame.append method is deprecated and will be removed from pandas in a future version. Use pandas.concat instead.`

Thanks.

**Comment by paudano** [comment]
I see, it was removed. I will fix this and push and update. It may not be until next week. I know there are some other Pandas features that have become deprecated, and fixing all those is on my PAV TODO list.

**Comment by dgordon562** [comment]
Pete:  this is not just a problem with the pandas deprecation of append.  Please see my issue: https://github.com/EichlerLab/pav/issues/41#issue-1837429475

**Comment by paudano** [comment]
I think this is the same bug as #41. It's fixed and will be out in 2.3.1, sorry for the delay!
